README.md

```diff
@@ -3,6 +3,10 @@
 This is a simple microservice that shows how a web endpoint can be provided
 for querying role IDs from an OMERO database.
```

Comment:
I know this is not part of your PR, but I noticed today that this repo still has a placeholder description which it would be good to update ASAP.